packages/features/webhooks/lib/cron.ts

```diff
@@ -27,6 +27,9 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       await fetch(job.subscriberUrl, {
         method: "POST",
         body: job.payload,
+        headers: {
+          "Content-Type": "application/json",
```

Comment:
If payload is not json we should add `application/x-www-form-urlencoded` here instead

Same as here: https://github.com/calcom/cal.com/blob/main/packages/features/webhooks/lib/sendPayload.ts#L108




<br class="Apple-interchange-newline">